app/models/mfa_copy/required_popular_gems.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class MfaCopy::RequiredPopularGems
```

Comment:
Is this being used anywhere?

Reply:
Nope not anymore! I have to delete this too.